org.eclipse.scout.rt.platform/src/main/java/org/eclipse/scout/rt/platform/context/RunContext.java

```diff
@@ -330,8 +331,12 @@ public RunContext withTransactionScope(final TransactionScope transactionScope)
   }
 
   /**
+   * @deprecated Use <code>ITransaction.CURRENT.get()</code> to access the current transaction. This method is only used
```

Comment:
> This method is only used at build time of a {@link RunContext} when a specific transaction is to be used by a RunContext.

This statement seems to apply to withTransaction(), not getTransaction().

Maybe reformulate:
`This member represents the transaction that was set during the build time of a run context, either explicitly or via {@link copy()}. It may differ from the actual transaction used during the execution of the runnable, e.g. when the transaction scope {@link TransactionScope.REQUIRES_NEW} is used.`

Reply:
done